air/src/options.rs

```diff
@@ -291,6 +300,10 @@ impl Deserializable for ProofOptions {
             FieldExtension::read_from(source)?,
             source.read_u8()? as usize,
             source.read_u8()? as usize,
+            match source.read_u8()? {
+                0 => BatchingType::Linear,
+                _ => BatchingType::Algebraic,
```

Comment:
I would do exact matching - e.g., if it's not `0` or `1`, let's return an error.